utils/test/src/jvmMain/kotlin/SystemEnvironmentExtensions.kt

```diff
@@ -0,0 +1,146 @@
+/*
+ * Copyright (C) 2016 Stephen Samuel
+ * Copyright (C) 2025 The ORT Project Authors (see <https://github.com/oss-review-toolkit/ort/blob/main/NOTICE>)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+// This file is based on
+// https://github.com/kotest/kotest/blob/v5.9.1/kotest-extensions/src/jvmMain/kotlin/io/kotest/extensions/system/SystemEnvironmentExtensions.kt.
+
+package org.eclipse.apoapsis.ortserver.utils.test
```

Comment:
IIRC the same was done for ORT, can we reuse this from the ORT test utils after the next ORT release?

Reply:
Not straight as-is... the version in ORT was slightly trimmed down and lacks the `OverrideMode` paramater as ORT does not use it, but ORT Server does. So for simplicity I decided to use another copy here. But we could also extend the ORT copy and then use it here.

Reply:
Yes, I think it would be nice to only have to maintain one copy.

Reply:
I saw this too late. I'll take care of enhancing the ORT version for the server's needs, and then taking that into use with the next release.

Reply:
See https://github.com/oss-review-toolkit/ort/pull/10792.

Reply:
And now https://github.com/eclipse-apoapsis/ort-server/pull/3440.